lib/alephant/broker/response/factory.rb

```diff
@@ -4,12 +4,12 @@ module Alephant
   module Broker
     module Response
       class Factory
-        def self.response_for(request)
+        def self.response_for(request, env)
```

Comment:
Yup passing in the env sounds sensible, but i'd maybe re-name the variable to `request_env` just so it can't be confused with the response env